src/varlinkctl/varlinkctl.c

```diff
@@ -522,6 +522,7 @@ static int reply_callback(
         if (!arg_quiet)
                 sd_json_variant_dump(parameters, arg_json_format_flags, stdout, NULL);
 
+        fflush(stdout);
```

Comment:
this patch can be dropped, ebc8b9e45b11c8e6c66b91e36cba6591ad6f9b38 already addresses that.